src/components/Countdown.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from "react";
+
+interface CountdownProps {
+  time: number; // in seconds
```

Comment:
```suggestion
  time?: number; // in seconds
```

Reply:
Didn't we decide earlier that defaultProps should not be optional?